### ISSUE TITLE
[Bug] panic: runtime error: negative shift amount

### ISSUE THREAD

**Issue by andorsk** [TBD54566975/ssi-service#227]
**Describe the bug**
```
➜  ssi-service git:(main) mage spec
panic: runtime error: negative shift amount

goroutine 1 [running]:
github.com/swaggo/swag.(*PackageDefinitions).evaluateConstValue(0x1400106a140, 0x10?, 0x14002d69468?, {0x104d25298?, 0x14001092fc0?}, {0x104d238c0, 0x140001d1800}, 0xaa?)
	/Users/akmb2/go/pkg/mod/github.com/swaggo/swag@v1.8.8/package.go:146 +0xdcc
github.com/swaggo/swag.(*PackageDefinitions).evaluateConstValue(0x1400106a140, 0x10cc3d338?, 0x105288108?, {0x104d25808?, 0x14001097d80?}, {0x104d238c0, 0x140001d1800}, 0x0?)
	/Users/akmb2/go/pkg/mod/github.com/swaggo/swag@v1.8.8/package.go:151 +0xd3c
github.com/swaggo/swag.(*PackageDefinitions).evaluateConstValue(0x1400106a140, 0x14006998ff0?, 0x140067218f0?, {0x104d25298?, 0x14001092ff0?}, {0x104d238c0, 0x140001d1800}, 0x140013eb8b8?)
	/Users/akmb2/go/pkg/mod/github.com/swaggo/swag@v1.8.8/package.go:115 +0xa0
github.com/swaggo/swag.(*PackagesDefinitions).EvaluateConstValue(0x14002d69668?, 0x1400106a140, 0x140055e6dc0, 0x0)
	/Users/akmb2/go/pkg/mod/github.com/swaggo/swag@v1.8.8/packages.go:282 +0x144
github.com/swaggo/swag.(*PackagesDefinitions).evaluateAllConstVariables(...)
	/Users/akmb2/go/pkg/mod/github.com/swaggo/swag@v1.8.8/packages.go:265
github.com/swaggo/swag.(*PackagesDefinitions).ParseTypes(0x140001d1800)
	/Users/akmb2/go/pkg/mod/github.com/swaggo/swag@v1.8.8/packages.go:110 +0x1d0
github.com/swaggo/swag.(*Parser).ParseAPIMultiSearchDir(0x1400028a000, {0x140002880e0?, 0x1?, 0x0?}, {0x16b6c32c5?, 0xb?}, 0x3)
	/Users/akmb2/go/pkg/mod/github.com/swaggo/swag@v1.8.8/parser.go:375 +0x29c
github.com/swaggo/swag/gen.(*Gen).Build(0x14000253920, 0x1400024c2a0)
	/Users/akmb2/go/pkg/mod/github.com/swaggo/swag@v1.8.8/gen/gen.go:182 +0x4dc
main.initAction(0x14000278780?)
	/Users/akmb2/go/pkg/mod/github.com/swaggo/swag@v1.8.8/cmd/swag/main.go:158 +0x698
github.com/urfave/cli/v2.(*Command).Run(0x1400022ab40, 0x14000276300)
	/Users/akmb2/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/command.go:163 +0x4fc
github.com/urfave/cli/v2.(*App).RunContext(0x1400024e4e0, {0x104d262b0?, 0x140001ae008}, {0x140001b6000, 0xb, 0xb})
	/Users/akmb2/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:313 +0x860
github.com/urfave/cli/v2.(*App).Run(...)
	/Users/akmb2/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:224
main.main()
	/Users/akmb2/go/pkg/mod/github.com/swaggo/swag@v1.8.8/cmd/swag/main.go:229 +0x65c
Error: running "swag init -g cmd/main.go --pd -o doc -ot yaml --parseDepth=3 --parseGoList=false" failed with exit code 2
```
**To Reproduce**
*Steps to reproduce the behavior:*
`mage spec` caused this

**Expected behavior**
Generates the openapi specs. No error.

**Environment (please complete the following information):**
 - OS: OSX M1
 - Version: SHA 8349729feb82ea81afc8f762283c80c93ded0048

**Additional context**

**Comment by andorsk** [comment]
seems related to https://github.com/swaggo/swag/issues/1309

**Comment by andorsk** [comment]
Fixed downgraded to `v1.8.6`: go install github.com/swaggo/swag/cmd/swag@v1.8.6
